models/public/efficientdet-d1-tf/model.yml

```diff
@@ -0,0 +1,121 @@
+  - name: backbone/efficientnet_model.py
+    size: 27419
+    sha256: 0a488e70c46bfa6d98fec66b4c518e0b3a25eedd8ca836123f4ef335dd12bf0c
+    source: https://raw.githubusercontent.com/google/automl/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/backbone/efficientnet_model.py
+  - name: backbone/backbone_factory.py
+    size: 2963
+    sha256: 3babe95f1bd104fdb1f54e49e68e2a12478933058d6a09c2a46401e80fab06c9
+    source: https://raw.githubusercontent.com/google/automl/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/backbone/backbone_factory.py
+  - name: keras/anchors.py
+    size: 9175
+    sha256: a88000a453e1ec1194cff2155a1148ef1c2ca87ad563cff2f412f1a597c3ffd7
+    source: https://raw.githubusercontent.com/google/automl/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/keras/anchors.py
+  - name: keras/util_keras.py
+    size: 2198
+    sha256: 81e21202054018d7616ab1d0f41eceaf8b4e31a6d6ee88cb53c251abea9fa582
+    source: https://raw.githubusercontent.com/google/automl/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/keras/util_keras.py
+  - name: keras/fpn_configs.py
+    size: 6120
+    sha256: 245cdd3308141fbb6d2b04b850d45fabb56a0f8ac74c857c7dd0d38238664f0d
+    source: https://raw.githubusercontent.com/google/automl/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/keras/fpn_configs.py
+  - name: keras/postprocess.py
+    size: 17677
+    sha256: 7ca776128887a813089dfc81a61cd4e82b4c3add100a5bafc1951f7dedd1fb4b
+    source: https://raw.githubusercontent.com/google/automl/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/keras/postprocess.py
+postprocessing:
+  # disable imports that aren't needed for this model and code that uses them
+  - $type: regex_replace
+    file: inference.py
+    pattern: '(import det_model_fn|from (keras import label_util|visualize)|from PIL)'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: hparams_config.py
+    pattern: 'import yaml|import ast'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: model_inspect.py
+    pattern: 'from PIL'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: backbone/backbone_factory.py
+    pattern: 'from backbone import efficientnet_lite_builder'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: dataloader.py
+    pattern: 'from (object_detection|keras|absl)'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: keras/anchors.py
+    pattern: 'from object_detection'
+    replacement: '# \g<0>'
+
+  - $type: unpack_archive
+    format: gztar
+    file: efficientdet-d1.tar.gz
+model_optimizer_args:
+  - --input_shape=[1,640,640,3]
+  - --input=convert_image/Cast
+  - --reverse_input_channels
+  - --input_model=$conv_dir/efficientdet-d1_saved_model/efficientdet-d1_frozen.pb
+  - --output=concat,concat_1
+  - --transformations_config=$mo_dir/extensions/front/tf/automl_efficientdet.json
```

Comment:
looks like we need update openvino package on CI before this model can be merged